relay/config.go

```diff
@@ -0,0 +1,44 @@
+package relay
+
+import core "github.com/Layr-Labs/eigenda/core/v2"
+
+// Config is the configuration for the relay Server.
+type Config struct {
+
+	// Shards contains the IDs of the relays that this server is willing to serve data for. If empty, the server will
+	// serve data for any shard it can.
+	Shards []core.RelayKey
+
+	// MetadataCacheSize is the size of the metadata cache. Current cache implementation is unaware of data sizes, and
```

Comment:
This and below: can just say it's the max num of entries/items in the the cache (if there is limit for bytes, it can be named properly as XXSizeBytes)

Reply:
updated